guide/source/program_administrator/opportunities_benefits.rst

```diff
@@ -15,7 +15,7 @@ industry:
 Once adopted, HPXML has the potential to dramatically lower costs in a wide
 range of areas because software vendors can use off-the-shelf solutions that
 they have already developed, rather than building new systems for each client.
-If this is done in coordination with other HPXML jurisdiction, it can not only
+If this is done in coordination with other HPXML jurisdiction [WHAT IS AN HPXML JURISDICTION? SHOULD THIS SAY PROGRAM JURISDICTION? IF NOT, HPXML JURISDICTION NEEDS TO BE DEFINED.], it can not only
```

Comment:
@nmerket yes we are talking utility or local government program in the majority case.  I will clean this up to state "program".